$NetBSD$

Compile 'strip' test with CFLAGS, so -isysroot can be passed on OSX.
Correct library detection on NetBSD.
Fix version checking of ICU 49 and above.

--- config.lib.orig	2012-04-15 11:55:05.000000000 +0000
+++ config.lib
@@ -2001,7 +2001,7 @@ check_strip() {
 	elif [ "$os" = "OSX" ]; then
 		# Most targets have -V in strip, to see if they exists... OSX doesn't.. so execute something
 		echo "int main(int argc, char *argv[]) { }" > strip.test.c
-		$cxx_host strip.test.c -o strip.test
+		$cc_host $CFLAGS strip.test.c -o strip.test
 		check_compiler "host strip" "strip" "$host" "$strip" "$STRIP" "strip" "strip" "3" "strip.test"
 		rm -f strip.test.c strip.test
 	else
@@ -2533,7 +2533,7 @@ detect_library() {
 		if [ -z "$res" ]; then
 			log 2 "  trying /opt/local/include/$4$5... no"
 		fi
-		if [ -z "$res" ] && [ "$os" = "NETBSD" ]; then
+		if [ -z "$res" ]; then
 			eval "$2=`ls -1 /usr/pkg/include/$4*.h 2>/dev/null | egrep \"\/$5\$\"`"
 			eval "res=\$$2"
 			if [ -z "$res" ]; then
@@ -2864,7 +2864,7 @@ detect_icu() {
 
 	version=`$icu_config --version 2>/dev/null`
 	ret=$?
-	shortversion=`echo $version | cut -c 1,3`
+	shortversion=`echo $version | cut -c 1,2`
 	log 2 "executing $icu_config --version"
 	log 2 "  returned $version"
 	log 2 "  exit code $ret"
